content/metadata-stats/stats.textile

```diff
@@ -27,39 +27,91 @@ redirect_from:
   - /realtime/versions/v0.8/statistics
 ---
 
-Ably provides statistics at one minute intervals. Statistics are aggregated by minute, hour, day and month. They are available as:
+Ably provides statistics at one minute intervals. They are aggregated by minute, hour, day and month. The most recent statistics can be delayed by up to six seconds.
 
-* "Account-level statistics":#account covering all applications in your account
-* "App-level statistics":#app for each individual application
+Statistics are available as:
 
-Account-level and app-level statistics can also be retrieved from the "Control API.":/account/control-api
+* "Account statistics":#account covering all applications in your account
+* "App statistics":#app for each individual application
 
-<aside data-type='note'>
-<p>The most recent statistics are delayed by up to six seconds.</p>
-</aside>
+h2(#account). Account statistics
 
-h2(#account). Account-level statistics
+Account statistics provide information about all applications in your account.
 
-Account-level statistics are available as graphs, as tabular data, and for download from your "application dashboard":https://ably.com/accounts/any/usage_history. They are used to calculate your monthly quota and any potential "overages":https://faqs.ably.com/what-happens-if-i-exceed-my-package-connection-channel-or-message-limits.
+You can retrieve account statistics from:
 
-h2(#app). App-level statistics
+* The "Control API":#account-control
+* Your "account dashboard":#account-dashboard
+
+h3(#account-control). Account statistics via API
+
+You can retrieve account statistics using the "Control API":/account/control-api by making a call to the @accounts@ endpoint using your @accountId@ and @accessToken@:
```

Comment:
is this now recommended over the `rest.ably.io/stats` endpoint? if the old point isn't totally redundant now is it worth mentioning it here?

Reply:
Mmm, good point. Both are still valid as far as I know, so I'll add the REST API here.